src/solvers/dg/1d/dg.jl

```diff
@@ -94,7 +94,7 @@ function Dg1D(equation::AbstractEquations{NDIMS, NVARS}, surface_flux_function,
   end
 
   # Initialize boundary conditions
-  boundary_conditions = init_boundary_conditions(n_boundaries_per_direction, mesh)
+  boundary_condition = init_boundary_condition(n_boundaries_per_direction, mesh)
```

Comment:
Same argument here about plural boundary condition**s**

Reply:
That's old code that will be removed - I didn't want to touch this old code... Shall I?

Reply:
Fixed in 2d1fc09.